test/models/concerns/reminders/states/done_test.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Reminders::States::DoneTest < ActiveSupport::TestCase
+  include ActionMailer::TestHelper
+
+  test 'notify do nothing' do
+    reminder = reminders :reminder_of_ls_on_atahualpa_not_well
+    state    = Reminders::States::Done.new
+
+    state.notify reminder
+
+    refute reminder.state_class_type_change
+    assert_enqueued_emails 0
```

Comment:
```suggestion
    assert_no_emails
```
O bien puede ser como bloque, que creo que me gusta aún más. Sería algo como:
```ruby
assert_no_emails do
  state.notify reminder
end
```

Reply:
PD: lo mismo con todos los demas

Reply:
Ahh claro, tenés razón, si, vamos con `assert_no_enqueued_emails` nomás =)

Reply:
va ese nomas